src/lib/components/NewTargetOfEvaluation.svelte

```diff
@@ -41,6 +45,6 @@ function submit() {
 		</Input>
 	</FormGroup>
 
-	<Button color={'primary'} type="submit">Add</Button>
-	<Button color={'danger'}>Cancel</Button>
+	<Button color={'primary'} on:click={add}>Add</Button>
```

Comment:
any specific reason for not using the standard submit flow? This has the advantage that you can also use enter to submit for example.

Reply:
Yes, because the cancel button did not work and the controls were added.

Reply:
Not quite sure if I understand It correctly. the cancel button was missing the on:click yes, but the add button should use type=submit in my opinion to just submit the form.